packages/aws-cdk-lib/aws-stepfunctions/lib/states/wait.ts

```diff
@@ -16,7 +16,18 @@ export class WaitTime {
   public static duration(duration: cdk.Duration) { return new WaitTime({ Seconds: duration.toSeconds() }); }
 
   /**
-   * Wait until the given ISO8601 timestamp
+   * Wait for a number of seconds stored in the state object from string.
+   * This method can use JSONata expression.
+   *
+   * If you want to use fixed value, we recommend using `WaitTime.duration()`
+   *
+   * Example value: `{% $waitSeconds %}`
+   */
+  public static seconds(seconds: string) { return new WaitTime({ Seconds: seconds }); }
```

Comment:
For my understanding, is this method because JSONata expression is different vs JSONPath? `Seconds` vs `SecondsPath`.

Reply:
Yeah, `SecondsPath` is for JSONPath only property. In JSONata, we use `Seconds`. However, currently we can't set seconds using string type, so I added new seconds method. It can use JSONata expression. If this interface is foolish, I can add `jsonataExpression()` or etc. What do you think?
https://docs.aws.amazon.com/step-functions/latest/dg/state-wait.html

Reply:
outsider view here, but `.seconds` makes more sense than `secondsJsonataExpression` especially if AWS intends for JSONata to be the default going forward.